extra/utils/validation.js

```diff
@@ -0,0 +1,55 @@
+
+const users=[
+    {
+        traineeEmail: 'trainee1@successive.tech',
+        reviewerEmail: 'reviewer1@successive.tech',
+    },
+    {
+        traineeEmail: 'trainee2@successive.tech',
+        reviewerEmail: 'reviewer2@successive.tech',
+    },
+
+    {
+        traineeEmail: 'trainee3@successive.tech',
+        reviewerEmail: 'reviewer3@successive.t',
+    }
+]
+
+function validateEmail(email)
+{
+    const re = /^(([^<>()\[\]\.,;:\s@\"]+(\.[^<>()\[\]\.,;:\s@\"]+)*)|(\".+\"))@successive.tech$/;
+    return re.test(email);
+}
+
+function validateUsers(users)
+{
+    let validUsers=0;
```

Comment:
Declaring variables for valid and invalid count is unnecessary you can take advantage of .length property of array to get the no of record in array. 
For example to get valid user count you can simply write `userValid.length`